### ISSUE TITLE
GAMA 1.9.1 deb package is broken on Ubuntu 22.04

### ISSUE THREAD

**Issue by axb21** [gama-platform/gama.old#3835]
**Describe the bug**
I downloaded the file `GAMA_1.9.1_Linux_with_JDK.deb` from the GAMA website by visiting https://gama-platform.org/download and clicking the "Linux" button near the top of the page. I attempted to install this DEB package using `dpkg`. GAMA was *not* installed. `dpkg` reported:
```
chmod: cannot access '/opt/gama-platform': No such file or directory
chmod: cannot access '/opt/gama-platform/Gama': No such file or directory
chmod: cannot access '/opt/gama-platform/headless/gama-headless.sh': No such file or directory
```

**To Reproduce**
1. Visit https://gama-platform.org/download
2. Click on the button labeled "Linux" near the top right of that page
3. A download process should start, resulting in a file GAMA_1.9.1_Linux_with_JDK.deb
4. Double-click that file, or use `dpkg -i` to install it
5. Observe that GAMA is not installed, and that the install process issues "No such file or directory" errors

**Expected behavior**
The install process should complete with error, and GAMA 1.9.1 should be installed on the system and usable.

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04
 - PC Model: N/A (?)
 - GAMA version: 1.9.1
 - Java version: bundled with the installer
 - Graphics cards / Display system: N/A

**Additional context**

**Comment by axb21** [comment]
As an aside, the corresponding ZIP file works fine. I can run GAMA as expected. It seems the issue is isolated to the DEB packaging.

**Comment by RoiArthurB** [comment]
Very strange,

I just tried it on a new system and everything works fine : 

```
root@e9f0b20aaca7:/# neofetch 
            .-/+oossssoo+/-.               root@e9f0b20aaca7 
        `:+ssssssssssssssssss+:`           ----------------- 
      -+ssssssssssssssssssyyssss+-         OS: Ubuntu 22.04.2 LTS x86_64 
    .ossssssssssssssssssdMMMNysssso.       Host: ROG Strix G614JI_G614JI 1.0 
   /ssssssssssshdmmNNmmyNMMMMhssssss/      Kernel: 6.4.3-arch1-2-g14 
  +ssssssssshmydMMMMMMMNddddyssssssss+     Uptime: 19 hours, 2 mins 
 /sssssssshNMMMyhhyyyyhmNMMMNhssssssss/    Packages: 231 (dpkg) 
.ssssssssdMMMNhsssssssssshNMMMdssssssss.   Shell: bash 5.1.16 
+sssshhhyNMMNyssssssssssssyNMMMysssssss+   Resolution: 1920x1080 
ossyNMMMNyMMhsssssssssssssshmmmhssssssso   CPU: 13th Gen Intel i7-13650HX (20) @ 4.700GHz 
ossyNMMMNyMMhsssssssssssssshmmmhssssssso   GPU: Intel Device a78b 
+sssshhhyNMMNyssssssssssssyNMMMysssssss+   GPU: NVIDIA 01:00.0 NVIDIA Corporation Device 2860 
.ssssssssdMMMNhsssssssssshNMMMdssssssss.   Memory: 8324MiB / 15613MiB 
 /sssssssshNMMMyhhyyyyhdNMMMNhssssssss/
  +sssssssssdmydMMMMMMMMddddyssssssss+                             
   /ssssssssssshdmNNNNmyNMMMMhssssss/                              
    .ossssssssssssssssssdMMMNysssso.
      -+sssssssssssssssssyyyssss+-
        `:+ssssssssssssssssss+:`
            .-/+oossssoo+/-.

root@e9f0b20aaca7:/# wget https://github.com/gama-platform/gama/releases/download/1.9.1/GAMA_1.9.1_Linux_with_JDK.deb
[...]
2023-07-21 02:30:50 (1.05 MB/s) - 'GAMA_1.9.1_Linux_with_JDK.deb' saved [403030392/403030392]

root@e9f0b20aaca7:/# apt install ./GAMA_1.9.1_Linux_with_JDK.deb 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'gama-platform-jdk' instead of './GAMA_1.9.1_Linux_with_JDK.deb'
The following NEW packages will be installed:
  gama-platform-jdk
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 0 B/403 MB of archives.
After this operation, 0 B of additional disk space will be used.
Get:1 /GAMA_1.9.1_Linux_with_JDK.deb gama-platform-jdk amd64 1.9.1 [403 MB]
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package gama-platform-jdk.
(Reading database ... 9011 files and directories currently installed.)
Preparing to unpack /GAMA_1.9.1_Linux_with_JDK.deb ...
Unpacking gama-platform-jdk (1.9.1) ...
Setting up gama-platform-jdk (1.9.1) ...

root@e9f0b20aaca7:/# bash /opt/gama-platform/headless/gama-headless.sh -version
******************************************************************
* GAMA version 1.9.1                                             *
* http://gama-platform.org                                       *
* (c) 2007-2023 UMI 209 UMMISCO IRD/SU & Partners                *
******************************************************************
Welcome to Gama-platform.org version GAMA 1.9.1
```

Can you verify the integrity of your deb file ? :thinking: 
```
root@e9f0b20aaca7:/# sha256sum GAMA_1.9.1_Linux_with_JDK.deb 
00b086169ea3ccfb32c1f900a0602e474a28dd045a3ee5c60ef0eb39524c1037  GAMA_1.9.1_Linux_with_JDK.deb
```

**Comment by axb21** [comment]
> bash /opt/gama-platform/headless/gama-headless.sh -version

Not to nitpick, but you did not follow the procedure I outlined--I downloaded the DEB from the web site, and used `dpkg` to install, not `apt`. :smirk: 

Even so, both my procedure and yours work for me now. I have no explanation. It's possible the download was corrupted somehow (though I was shown no warning about this).

I guess this should be closed, since GAMA downloads and installs just fine now.
